### DISCUSSION TITLE
Multiline List Item Description

### DISCUSSION THREAD

**Discussion by Flipez** [charmbracelet/bubbles#154] [Q&A] [answered]
Is it possible to configure the list in some way that it supports varying heights for item descriptions?

I have multiple items and some of them do have muliple lines which I set as a description, while scrolling over them and paginating between pages with different descriptions rendering is broken.

Thanks in advance!

**Comment by muesli** [comment, marked as answer]
Yes, you can set the height by customizing the `Height` of the `ItemDelegate`: https://github.com/charmbracelet/bubbles/blob/master/list/list.go#L43

**Reply by Flipez** [reply]
Hi @muesli,

It seems that https://github.com/charmbracelet/bubbles/pull/155 does partially solve this issue but it also seems that this is only possible to set the height via the delegate on the list itself, not on an (individual) item level - or do I miss something?

I'd like to set the height of an item based on its description content.

**Reply by bashbunni** [reply]
Hey @Flipez you might find this discussion to be helpful https://github.com/charmbracelet/bubbles/discussions/111
you can define your own [ItemDelegate](https://pkg.go.dev/github.com/charmbracelet/bubbles@v0.10.2/list#ItemDelegate) that allows you to customize the styling for list items as meowgorithm showed in this discussion
